internal/jitsi/jvb.go

```diff
@@ -392,7 +436,7 @@ func (j *JVB) getExternalIP() string {
 		case <-tick.C:
```

Comment:
Not sure that it will be a better solution for your case but it will be more idiomatic to: 
 1. create service
 2. set reference with [controllerutil#SetControllerReference](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#SetControllerReference). Then jvb reconciler will be called on service update.
 3. get service IP